feat: Add the ability to configure per-module color styles (#285)
starship/starship: docs/advanced-config/README.md

If you like the result, add these lines to your shell configuration file 
(`~/.bashrc` or `~/.zsrhc`) to make it permanent.

## Style Strings

Style strings are a list of words, separated by whitespace. The words are not case sensitive (i.e. `bold` and `BoLd` are considered the same string). Each word can be one of the following:

  - `bold`
  - `underline`
  - `bg:<color>`
  - `fg:<color>`
  - `<color>`

> where `<color>` is a color specifier (discussed below). The last two are currently considered equivalent, though this may change in the future. The order of words in the string does not matter. 

Reviewer: To help avoid a mistake where we add an option to the end of the list, I think it's best if we reword this.

```suggestion
where `<color>` is a color specifier (discussed below). Both `fg:<color>` and `<color>` are currently considered equivalent, though this may change in the future. The order of words in the string does not matter. 
```

Author: Agreed.
